### PR TITLE
Fix: stETH on Optimism

### PR DESCRIPTION
**Pull request by TheDZhon**
**NB:** Please don't merge till the [vote 179 gets enacted](https://vote.lido.fi/vote/179) :warning: 

---

Context: 
- https://research.lido.fi/t/steth-on-optimism-upgrade-announcement-and-action-plan/8474
- https://research.lido.fi/t/lip-22-steth-on-l2/6855

Changes:
- [x] Remove proposed marks
- [x] Update Lido tokens integration guide
- [x] Update rollup bridging guide

### CODE REVIEW

File: docs/guides/lido-tokens-integration-guide.md [hunk 1]
```diff
@@ -279,12 +281,22 @@ Currently, wstETH token is [present on](/docs/deployed-contracts/index.md#lido-m
 with bridging implemented via [the canonical bridges recommended approach](/docs/token-guides/wsteth-bridging-guide.md).
 
 :::note
-Unlike on the Ethereum mainnet, wstETH for Lido Multichain is a plain ERC-20 token and cannot be unwrapped to unlock stETH on the corresponding L2 network as of now.
+Unlike on the Ethereum mainnet, wstETH for Lido Multichain is a plain ERC-20 token and cannot be unwrapped to unlock stETH on the corresponding L2 network unless handled via [LIP-22](https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-24.md)
```
Comment (tamtamchik) [inline docs/guides/lido-tokens-integration-guide.md:284]: LIP 22 or 24?

File: docs/guides/lido-tokens-integration-guide.md [hunk 2]
```diff
@@ -279,12 +281,22 @@ Currently, wstETH token is [present on](/docs/deployed-contracts/index.md#lido-m
...
 :::
 
 Without the shares bookkeeping, the token cannot provide the `wstETH/stETH` rate and the rewards accrued on-chain.
 Use the [wstETH/stETH rate feeds](/guides/lido-tokens-integration-guide.md#integration-utilities-rate-and-price-feeds) listed above.
 
+#### stETH (Optimism pilot)
+
+Effective 2024 October, stETH token has become available [on Optimism](/docs/deployed-contracts/index.md#optimism).
+The wstETH and stETH tokens design follows the [LIP-22](https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-24.md) architecture approach.
```
Comment (tamtamchik) [inline docs/guides/lido-tokens-integration-guide.md:293]: Same here? LIP-22 or 24?

File: docs/token-guides/wsteth-bridging-guide.md [hunk 3]
```diff
@@ -10,6 +10,11 @@ This document is intended for the developers representing network/rollup foundat
 
 :::info
 This guide doesn't cover yet bridging of rebasable stETH token nor bridging to non-L2-rollup networks. However, please note that bridging rebasable stETH token in a regular way might cause a loss of user assets due to the rewards accrued being stuck on an L1 bridge.
+
+Effective 2024 October, stETH token has become available [on Optimism](/docs/deployed-contracts/index.md#optimism).
+The wstETH and stETH tokens design follows the [LIP-22](https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-24.md) architecture approach.
```
Comment (tamtamchik) [inline docs/token-guides/wsteth-bridging-guide.md:15]: Same here, should be 22

**Review by tamtamchik — CHANGES_REQUESTED** [review]
Need some update for 👄 LIPs

File: docs/guides/lido-tokens-integration-guide.md (hunk 1 above)
Comment (TheDZhon) [reply docs/guides/lido-tokens-integration-guide.md]: my bad

**Review by grstepanov — APPROVED** [review]
lgtm

**Review by tamtamchik — APPROVED** [review]
lgtm

**Review by kovalgek — APPROVED** [review]

**Review by arwer13 — APPROVED** [review]

**Pull request merged**